packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -453,10 +453,67 @@ class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
   }
 }
 
-class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
-  _ModalBottomSheetRoute({
-    this.builder,
-    required this.capturedThemes,
+/// A route that represents a Material Design modal bottom sheet.
+///
+/// {@template flutter.material.ModalBottomSheetRoute}
+/// A modal bottom sheet is an alternative to a menu or a dialog and prevents
+/// the user from interacting with the rest of the app.
+///
+/// A closely related widget is a persistent bottom sheet, which shows
+/// information that supplements the primary content of the app without
+/// preventing the user from interacting with the app. Persistent bottom sheets
+/// can be created and displayed with the [showBottomSheet] function or the
+/// [ScaffoldState.showBottomSheet] method.
+///
+/// The `isScrollControlled` parameter specifies whether this is a route for
```

Comment:
```suggestion
/// The [isScrollControlled] parameter specifies whether this is a route for
```

(so dartdoc links to it, since its an actual property in the class)